### PR TITLE
[SYCL] Separate opencl specific event constructs in the lit test.

### PR DESCRIPTION
**Pull request by rbegam**
Signed-off-by: rbegam <rehana.begam@intel.com>

### CODE REVIEW

**Comment by bader** [comment]
The change looks good to me, but CI failure is concerning.

@romanovvlad, @vladimirlaz, any idea about the root cause of this failure?

> $ "env" "SYCL_DEVICE_TYPE=GPU" "SYCL_BE=PI_CUDA" "/localdisk2/sycl_ci/buildbot/worker/Lit_With_Cuda/llvm.obj/tools/sycl/test/basic_tests/Output/event_profiling_info.cpp.tmp.out"
> 
> event_profiling_info.cpp.tmp.out: /localdisk2/sycl_ci/buildbot/worker/Lit_With_Cuda/llvm.src/sycl/test/basic_tests/event_profiling_info.cpp:37: int main(): Assertion `Submit <= Start' failed.
>

**Comment by romanovvlad** [comment]
> 
> 
> The change looks good to me, but CI failure is concerning.
> 
> @romanovvlad, @vladimirlaz, any idea about the root cause of this failure?
> 
> > $ "env" "SYCL_DEVICE_TYPE=GPU" "SYCL_BE=PI_CUDA" "/localdisk2/sycl_ci/buildbot/worker/Lit_With_Cuda/llvm.obj/tools/sycl/test/basic_tests/Output/event_profiling_info.cpp.tmp.out"
> > event_profiling_info.cpp.tmp.out: /localdisk2/sycl_ci/buildbot/worker/Lit_With_Cuda/llvm.src/sycl/test/basic_tests/event_profiling_info.cpp:37: int main(): Assertion `Submit <= Start' failed.

No ideas.

**Comment by vladimirlaz** [comment]
the failure unrelated to the change as it affects only LIT and different test has failed.
I have no idea about the root cause. BTW: the test has been XFAILED in https://github.com/intel/llvm/commit/6d34b95c3eb3bccf5d739d66a7c57eed196cc206

**Comment by bader** [comment]
> the failure unrelated to the change as it affects only LIT and different test has failed.
> I have no idea about the root cause. BTW: the test has been XFAILED in [6d34b95](https://github.com/intel/llvm/commit/6d34b95c3eb3bccf5d739d66a7c57eed196cc206)

The XFAIL condition there is wrong. See https://github.com/intel/llvm/commit/6d34b95c3eb3bccf5d739d66a7c57eed196cc206#r42033961.

File: sycl/test/basic_tests/event.cpp [hunk 1]
```diff
@@ -1,5 +1,3 @@
-// REQUIRES: opencl || level0
-
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out -L %opencl_libs_dir -lOpenCL
```
Comment (vladimirlaz) [inline sycl/test/basic_tests/event.cpp:1]: You can remove -lOpenCL because thre eare no OpenCL direct calls in the source anymore.

**Review by vladimirlaz — APPROVED** [review]

File: sycl/test/basic_tests/event.cpp (hunk 1 above)
Comment (bader) [inline sycl/test/basic_tests/event.cpp:1]:
```suggestion
// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

**Review by v-klochkov — APPROVED** [review]

**Review by vladimirlaz — APPROVED** [review]

**Pull request merged**